src/Gui/DlgUnitsCalculator.ui

```diff
@@ -114,12 +114,17 @@ To add a calculation press Return in the value input field</string>
       <item row="0" column="2">
        <widget class="QLabel" name="label_6">
         <property name="text">
-         <string>Scheme:</string>
+         <string>Unit System:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
-       <widget class="QComboBox" name="comboBoxScheme"/>
+       <widget class="QComboBox" name="comboBoxScheme">
+        <property name="toolTip">
+         <string>Unit system to be used to the Quantity
+The preference system is the one set in the FreeCAD preferences</string>
```

Comment:
We should avoid using the name FreeCAD because that breaks the branding of the software. We want it to be brand-neutral.

`The preference system is the one set in the general preferences for the program</string>`